apps/dcos_overlay/src/dcos_overlay_lashup_kv_listener.erl

```diff
@@ -136,3 +156,31 @@ to_str(Mac) when length(Mac) =:= 6 ->
     lists:flatten(string:join(List, ":"));
 to_str(IP) ->
     inet:ntoa(IP).
+
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, lashup_events_received_total},
+        {help, "Total number of Lashup events received."}]),
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, reconciliations_total},
+        {help, "Total number of overlay configuration reconciliations attempted."}]),
```

Comment:
What kind of information can we get from this metric? It does reconciliation every 5 minutes anyway.